openedx/core/djangoapps/user_authn/views/tests/test_login.py

```diff
@@ -284,6 +287,19 @@ def test_login_ratelimited(self):
         response, _audit_log = self._login_response('test@edx.org', 'wrong_password')
         self._assert_response(response, success=False, value='Too many failed login attempts')
 
+    def test_login_refresh(self):
+        def _assert_jwt_cookie_present(response):
+            self.assertEqual(response.status_code, 200)
+            self.assertIn(jwt_cookies.jwt_refresh_cookie_name(), self.client.cookies)
+
+        setup_login_oauth_client()
+        with JWT_COOKIES_FLAG.override(True):
+            response, _ = self._login_response('test@edx.org', 'test_password')
+            _assert_jwt_cookie_present(response)
+
+            response = self.client.post(reverse('login_refresh'))
+            _assert_jwt_cookie_present(response)
```

Comment:
Any need to verify the cookie is different?

Reply:
We have another lower-level test (in cookies.py) that checks for that - so I didn't duplicate that test here at the view level.